frontend/plan/types.ts

```diff
@@ -1,8 +1,15 @@
-export enum School {
+export enum School { // TODO: need to add more here?
     SEAS = "SEAS",
     WHARTON = "WH",
     COLLEGE = "SAS",
     NURSING = "NURS",
+    LPS = "LPS",
+    DESIGN = "DSGN",
+    GSE = "GSE",
+    LAW = "LAW",
+    MEDICINE = "MED",
+    GRADE_MODE = "MODE", // this isn't really a school, but it's used for course attributes
```

Comment:
Is this something we could split using union types? or is it better left as a one-off? Would love to hear more background behind this decision!

Reply:
2 reasons
1. blindly following the conventions of the rest of this file (it was probably written before union types?)
2. easier to guarantee exhaustive matching (?)

I'm not a typescript expert, so I'll defer to you